README.md

```diff
@@ -33,16 +33,28 @@ An **Image Search** App to find **Anime** Details related to the Image. Created
 - View and Manage History of all searches
 - Customize App Behaviour from Settings Page
 
-## Support
+## Platform Support
 
 This package currently is only tested on android and the web platform.
+<details>
+  <summary>✅ android</summary>
+  
+  + [android-arm32.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-arm32-release.apk)
```

Comment:
Why does it use `+` instead of `-` here?

Reply:
Check Now